.github/workflows/release.yaml

```diff
@@ -1,33 +1,87 @@
+# GitHub release workflow.
+#
+# This workflow is a bit complicated because we have to build darwin binaries on
+# a mac runner, but the mac runners are extremely slow. So instead of running
+# the entire release on a mac (which will take an hour to run), we run only the
+# mac build on a mac, and the rest on a linux runner. The final release is then
+# published using a final linux runner.
 name: release
 on:
   push:
     tags:
       - "v*"
   workflow_dispatch:
+    inputs:
+      snapshot:
+        description: Perform a snapshot/dry-run release (required if the ref is not a tag)
+        type: boolean
+        required: true
```

Comment:
I think we should remove this for now!

Reply:
It doesn't actually create a release, this is just for testing the release workflow

Reply:
I improved the language around this input to explain that it will not create a github release